tests/online_migration/sle12_online_migration/snapper_rollback.pm

```diff
@@ -26,10 +26,22 @@ sub check_rollback_system() {
 sub run() {
     my $self = shift;
 
-    # login to snapshot and perform snapper rollback
-    assert_screen 'linux-login', 200;
+    # login to before online migration snapshot
+    # tty would not appear quite often after booting snapshot
```

Comment:
It does not appear because of [bsc#980337](https://bugzilla.suse.com/show_bug.cgi?id=980337) I guess?